prodigy_hf/textcat.py

```diff
@@ -43,7 +43,7 @@ def generator(examples) -> Iterable[Dict]:
             if (variant == "multi") and ex['accept']:
                 # It could be that the dataset was accepted but didn't have anything selected. 
                 label = label2id[ex["accept"][0]]
-            if label: 
+            if label is not None: 
```

Comment:
Although checking explicitly against `None` is more correct, this technically doesn't change anything as you'll still be getting just the first label on the list. This recipe only supports exclusive textcat so the `accept` is expected to be length one.  See the docs here: https://prodi.gy/docs/plugins#textcat-support